persistence-modules/hibernate-libraries/create-database.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+PWD=`pwd`
+docker run \
+  -p 53306:3306 \
+  --name=mysql57-hibernate-types \
+  -e MYSQL_ALLOW_EMPTY_PASSWORD=true \
+  -v $PWD/docker/etc/mysql/conf.d:/etc/mysql/conf.d \
```

Comment:
enclose in quotes:
"$PWD/docker/etc/mysql/conf.d":...
"$PWD/docker/docker-entrypoint-initdb.d":...